pkg/ottl/ottlfuncs/func_useragent.go

```diff
@@ -38,10 +38,13 @@ func userAgent[K any](userAgentSource ottl.StringGetter[K]) ottl.ExprFunc[K] { /
 			return nil, err
 		}
 		parsedUserAgent := parser.ParseUserAgent(userAgentString)
+		parsedOS := parser.ParseOs(userAgentString)
 		return map[string]any{
 			semconv.AttributeUserAgentName:     parsedUserAgent.Family,
 			semconv.AttributeUserAgentOriginal: userAgentString,
 			semconv.AttributeUserAgentVersion:  parsedUserAgent.ToVersionString(),
+			semconv.AttributeOSName:            parsedOS.Family,
+			semconv.AttributeOSVersion:         parsedOS.ToVersionString(),
```

Comment:
I think those attributes should be in the UserAgent namespace instead, but unluckily they are not available in semconv yet (https://github.com/open-telemetry/semantic-conventions/pull/1434).

Meanwhile, could we create them manually? 
```suggestion
			"user_agent." + semconv.AttributeOSName:            parsedOS.Family,
			"user_agent." +semconv.AttributeOSVersion:         parsedOS.ToVersionString(),
```

Reply:
Thanks @ioandr , FIXUP looks good to me. It seems I cannot "resolve the conversation" (no UI button), maybe because of the forced push? :thinking:

Reply:
Unless the semconv sig is waiting for proof-of-concept we should wait until the semconv sig as accepted them as experimental.